site/public/js/details.js

```diff
@@ -1,7 +1,9 @@
-/* global courseUrl */
+/* global courseUrl, showPopup */
 /* exported gradeableMessageAgree, gradeableMessageCancel, showGradeableMessage, hideGradeableMessage, expandAllSections, collapseAllSections, grade_inquiry_only, reverse_inquiry_only, inquiry_update */
 const MOBILE_BREAKPOINT = 951;
 
+const checkboxes = document.getElementsByClassName('toggle-columns-box');
```

Comment:
Is there a way to avoid putting a very generically-named variable in the global namespace?